draft-ietf-httpbis-sfbis.md

```diff
@@ -1056,12 +958,64 @@ The serialization algorithm is defined in a way that it is not strictly limited
 Implementations are allowed to limit the size of different structures, subject to the minimums defined for each type. When a structure exceeds an implementation limit, that structure fails parsing or serialization.
 
 
+# Informative ABNF {#abnf}
```

Comment:
Does the title need to say "Informative ABNF"? It already says it is non-normative below. How about just "ABNF Examples" or "ABNF Illustrations"?

Reply:
...or just "ABNF".

Reply:
That too =)